Thrive.csproj

```diff
@@ -13,6 +13,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <Deterministic>false</Deterministic>
+    <LangVersion>Latest</LangVersion>
```

Comment:
Instead of randomly going with whatever tool considers the latest version. I think explicit 7.3 version should be used (as I don't think mono works with C# 8, yet).
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

Reply:
Well then let's do 8.0 and see if that builds.